e2e-tests/helpers/date.js

```diff
@@ -19,3 +19,16 @@ export const createTimestampFromNumbers = (day, month, year) => new Date(`${mont
  * @returns {String} Formatted date
  */
 export const formatDate = (timestamp, dateFormat = DATE_FORMAT.DEFAULT) => format(new Date(timestamp), dateFormat);
+
+/**
+ * createDateOneMonthInThePast
+ * Create a date that is one month in the past.
+ * @returns {String} Date one month in the past
+ */
+export const createDateOneMonthInThePast = () => {
```

Comment:
just wondering if it is worth changing this to take a variable for any months in the past so we can use it in the future if needed for say 3 months etc

Reply:
Good idea, could change this in the future if required? Happy to change if a blocker though.